spec/system/candidate_interface/continuous_applications/submitting/candidate_submitting_application_spec.rb

```diff
@@ -18,7 +20,7 @@
     then_i_see_a_interruption_page_for_personal_statement
 
     when_i_continue_without_editing
-    then_i_should_see_the_review_and_submit_page
+    then_i_should_be_on_the_review_and_submit_page(application_choice: @application_choice)
```

Comment:
I would suggest we remove `should` from all methods. It's redundant and is not assertive.

Reply:
Also, we should be using the instance variable and not passing arguments to these methods unless it is somehow necessary

Reply:
I kinda of agree with the should, I consciously decided to keep the same pattern as we have many like that in the code:
```
⤷  ag then_i_should spec/ | wc -l
     850
```

Reply:
About the instance variable, I agree. I will change

Reply:
About the instance variable is done.